frontend/src/util/editorCommentUtils.js

```diff
@@ -43,6 +42,25 @@ function isCommentThreadIDMark(mayBeCommentThread) {
     return mayBeCommentThread.indexOf(COMMENT_THREAD_PREFIX) === 0;
 }
 
+export function insertMaybeComment (editor, selectedText, setMaybeComment) {
+    setMaybeComment(selectedText);
+    Editor.addMark(editor, 'isMaybeComment', true);
+}
+
+export function ifMaybeCommentOnTextNode(textnode) {
+    const keys = Object.keys(textnode)
```

Comment:
just curious but is there a more like idiomatic slate api for this? like getMarksFrom(node) or something like that instead of hunting for the object keys ourselves 0_o

Reply:
only editor.marks to get all marks i think